tests/TestUtils.hs

```diff
@@ -0,0 +1,9 @@
+{-# LANGUAGE CPP #-}
+
+module TestUtils where
+
+import Servant.Client
+
+#if !MIN_VERSION_servant_client(0,16,0)
+type ClientError = ServantError
+#endif
```

Comment:
This is a temporary measure until the up-coming upgrade to our Nix package set gets us through the "Servant hump" around version 0.16 of that library family.

Reply:
The GHC 8.6 upgrade should be coming very soon.  We're already working on it for pact-web.  So we may want to hold off merging this and just jump straight to the new version.  I'll defer to @slpopejoy and @emilypi to make the call though.